### PR TITLE
adding ranking baseline determ

### PR DESCRIPTION
**Pull request by yzhang123**
Signed-off-by: Yang Zhang <yangzhang@nvidia.com>

# What does this PR do ?

Add a one line overview of what this PR aims to accomplish.

**Collection**: [Note which collection this PR will affect]

# Changelog 
- Add specific line by line info of high level changes in this PR.

# Usage
* You can potentially add a usage example below

```python
# Add a code snippet demonstrating how to use this 
```

# Before your PR is "Ready for review"
**Pre checks**:
- [ ] Make sure you read and followed [Contributor guidelines](https://github.com/NVIDIA/NeMo/blob/main/CONTRIBUTING.md)
- [ ] Did you write any new necessary tests?
- [ ] Did you add or update any necessary documentation?
- [ ] Does the PR affect components that are optional to install? (Ex: Numba, Pynini, Apex etc)
  - [ ] Reviewer: Does the PR have correct import guards for all optional libraries?
  
**PR Type**:
- [ ] New Feature
- [ ] Bugfix
- [ ] Documentation

If you haven't finished some of the above items you can still open "Draft" PR.

## Who can review?

Anyone in the community is free to review the PR once the checks have passed. 
[Contributor guidelines](https://github.com/NVIDIA/NeMo/blob/main/CONTRIBUTING.md) contains specific people who can review PRs to various areas.

# Additional Information
* Related to # (issue)

### CODE REVIEW

File: nemo_text_processing/text_normalization/en/taggers/whitelist.py [hunk 1]
```diff
@@ -66,9 +73,6 @@ def _get_whitelist_graph(input_case, file, is_default=True):
 
         graph = _get_whitelist_graph(input_case, get_abs_path("data/whitelist_tts.tsv"))
 
-        # if lm:
```
Comment (ekmb) [inline nemo_text_processing/text_normalization/en/taggers/whitelist.py:69]: we need this for thresholding

File: nemo_text_processing/text_normalization/en/taggers/tokenize_and_classify_baseline.py [hunk 2]
```diff
@@ -0,0 +1,209 @@
...
+                pynutil.add_weight(time_graph, 1.1)
+                | pynutil.add_weight(decimal_graph, 1.1)
+                | pynutil.add_weight(measure_graph, 1.1)
+                | pynutil.add_weight(cardinal_graph, 1.1)
+                | pynutil.add_weight(ordinal_graph, 1.1)
+                | pynutil.add_weight(telephone_graph, 1.1)
+                | pynutil.add_weight(electronic_graph, 1.1)
+                | pynutil.add_weight(fraction_graph, 1.1)
+                | pynutil.add_weight(money_graph, 1.1)
+                | pynutil.add_weight(date_graph, 1.1)
```
Comment (ekmb) [inline nemo_text_processing/text_normalization/en/taggers/tokenize_and_classify_baseline.py:170]: date 1.09 to match default TN

File: nemo_text_processing/text_normalization/en/taggers/tokenize_and_classify_baseline.py [hunk 3]
```diff
@@ -0,0 +1,209 @@
...
+            # v_telephone_graph = vTelephone(deterministic=True).fst
+            # v_electronic_graph = vElectronic(deterministic=True).fst
+            # measure = vMeasure(decimal=decimal, cardinal=cardinal, fraction=fraction, deterministic=True)
+            # v_measure_graph = measure.fst
+            v_time_graph = vTime(deterministic=True).fst
+            # v_date_graph = vDate(ordinal=ordinal, deterministic=deterministic, baseline=True).fst
+            # v_money_graph = vMoney(decimal=decimal, deterministic=deterministic).fst
+            # v_roman_graph = vRoman(deterministic=deterministic, baseline=True).fst
+
+            time_final = pynini.compose(time_graph, v_time_graph)
```
Comment (ekmb) [inline nemo_text_processing/text_normalization/en/taggers/tokenize_and_classify_baseline.py:157]: is this one used?

**Review by ekmb — CHANGES_REQUESTED** [review]

File: nemo_text_processing/text_normalization/en/taggers/tokenize_and_classify_baseline.py (hunk 3 above)
Comment (yzhang123) [reply nemo_text_processing/text_normalization/en/taggers/tokenize_and_classify_baseline.py]: no deleted

File: nemo_text_processing/text_normalization/en/taggers/tokenize_and_classify_baseline.py (hunk 2 above)
Comment (yzhang123) [reply nemo_text_processing/text_normalization/en/taggers/tokenize_and_classify_baseline.py]: done, also updated whitelist to 1.01

File: nemo_text_processing/text_normalization/en/taggers/whitelist.py (hunk 1 above)
Comment (yzhang123) [reply nemo_text_processing/text_normalization/en/taggers/whitelist.py:69]: ok moved back comment

**Review by ekmb — APPROVED** [review]

**Pull request merged**